pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -381,7 +381,8 @@ func (m *managerImpl) reclaimNodeLevelResources(resourceToReclaim v1.ResourceNam
 		// attempt to reclaim the pressured resource.
 		reclaimed, err := nodeReclaimFunc()
 		if err != nil {
-			glog.Warningf("eviction manager: unexpected error when attempting to reduce %v pressure: %v", resourceToReclaim, err)
+			glog.Errorf("eviction manager: unexpected error when attempting to reduce %v pressure: %v", resourceToReclaim, err)
+			continue
```

Comment:
for deleteImages, we always report the amount for image gc even in the face of errors.

so i agree with @dashpole that this is not really possible with current code.

for future code, i would be fine just logging a warning, and ensure that reclaim if not set is 0.